Docs/user_guide/modules/volumes.md

```diff
@@ -54,6 +54,9 @@ in `Description` column in `Add data...` window. This only available for nrrd an
 
 ### Load a series of png, jpeg, or tiff images as volume
 
+Consider the [Image Stacks](https://github.com/SlicerMorph/S_2020/blob/master/Day_1/ImageStacks/ImageStacks.md) provided by the [SlicerMorph](https://slicermorph.github.io/) extension to work with large sets of high resolution images.  It allows you to automatically convert RGB to grayscale scalar volume, specify image spacing, and downsample large images.
+
+Alternatively you can load and manipulate the data directly using functionality in the core Slicer app:

```

Comment:
Alternatively you can load and manipulate the data directly using Slicer core modules: